uqcsbot/scripts/advent.py

```diff
@@ -291,20 +305,22 @@ def reply(message):
         reply('Error parsing leaderboard data.')
         raise
 
-    # whether to show full leaderboard for all days
-    full = not args.day
+    # whether to show only one day
+    day = args.day
+    # whether to use global points
+    globo = args.globo
+
     # header message
     message = f':star: *Advent of Code Leaderboard {args.code}* :trophy:'
-    if not full:
-        message += (
-            f'\n:calendar: *Day {args.day}* '
-            f'(sorted by {SORT_LABELS[args.sort]})'
-        )
+    if day:
+        message += f'\n:calendar: *Day {args.day}* (sorted by {SORT_LABELS[args.sort]})'
+    elif globo:
+        message += "Global Leaderboard Points"
```

Comment:
Can this be `\n:earth_asia: *Global Leaderboard Points*` to match the day leaderboard style?